src/services/textChanges.ts

```diff
@@ -462,6 +468,9 @@ namespace ts.textChanges {
             else if (isVariableDeclaration(node)) {
                 return { prefix: ", " };
             }
+            else if (isPropertyAssignment(node)) {
+                return { suffix: "," + this.newLineCharacter }

```

Comment:
@mhegazy some thing need help
in this case 
how should i add the `,` after every props  
if i update the `ObjectLiteralExpression`, the original `PropertyAssignment` will overship with this change